### DISCUSSION TITLE
Real Radio Broadcast transitions! no FadeIn and more... ?

### DISCUSSION THREAD

**Discussion by Radiotom** [AzuraCast/AzuraCast#5865] [Q&A]
In jears of Online Broadcasting and testing some Broadcast Automation Software I decided to test AzuraCast again after jears.

It's a powerfull Software that grow a lot in the last jears after my last using couple jears ago. Thanks for that 👍 the best feature is Stereotool integration. 

But... 

There are some Points that I don't understand why AC cannot do it right: 

- FadeIn? For what? It's not and mostly never nessesary for the transition of Songs. 
The problem is: When I set FadeIn Point as 0.00, Liquidsoap ignore the whole Fading.
Also the FadeOut of the Old Song. 🙄

Is there a way to say AC just fadeOut but never fade in and play the Old and New Song at End and Start of it overlapping? Maybe for 2 sec?

Notice: I don't care about Smart Fading because I don't like it. That's sometimes really to much information for a simple transition of two Songs. No metter how they start or end.*

- AutoCue? It seems that LS does not analys the Silence in Tracks of Start end End by Audiosingal to set Cue In and Cue out.
I cut to a view Songs silence of 10 Sec at the end and they will played every time? Why or is it not possible to do an AutoCue in an Out?

I am using Stereotool for Master Sound-Processing and my Songs are Normalized to -16 LUFS of EBU R128. In Stereotool is a little Compression that makes transitions better audible. I know that R128 Normalisation and Soundprocessing like Stereotool make no sense but there are a lot of advantage with it. 

So can anyone help be to disable the Crossfading with FadeIn without to disable the whole Fade Process? 

Thank u.

PS: AC is for me at the moment just a Fallback System and the Stream Central of 24/7 mAirList Live Automation of my Radiostation. But it would be nice to have the easiest things to overlap two songs without X Crossfading.

*Notice: X Crossfading only using People who mixes Beats together or doing Scratching Vinyls. But no one use that of real Radio Broadcasting. 😉

Last Thing: 
I know it's possible with LS because LibreTime does exact that right way of Fading: No FadeIn, only Fade Out if it's an HotEnd of Song. 😉 An overlap ist for the FadeOut Time of the Old Song.

**Comment by Radiotom** [comment]
Ok I think I resolved the main issue by editing the part of the Liquidsoap Config. 
![SmartSelect_20221121_210537_Chrome.jpg](https://user-images.githubusercontent.com/118381734/204006746-f28b6dfa-f3c3-4a7b-a629-847cc4bda034.jpg)

Sorry for my bad English.
In the logs are still the fadein parts of 3.00 but it sounds like the right way I like.

**Comment by Moonbase59** [comment]
Someone who knows what he’s talking about. Lovin’ it!
And thanks for the suggestion of the LS change.
